Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/vaccination/ui/details/VaccinationDetailsFragment.kt

```diff
@@ -84,37 +98,51 @@ class VaccinationDetailsFragment : Fragment(R.layout.fragment_vaccination_detail
             }
         }
 
+    private fun FragmentVaccinationDetailsBinding.bindToolbar() = toolbar.apply {
+        setNavigationOnClickListener { popBackStack() }
+        setOnMenuItemClickListener {
+            when (it.itemId) {
+                R.id.menu_covid_certificate_delete -> {
+                    DialogHelper.showDialog(deleteTestConfirmationDialog)
+                    true
+                }
+                else -> onOptionsItemSelected(it)
+            }
+        }
+    }
+
     private fun FragmentVaccinationDetailsBinding.bindCertificateViews(
         certificate: VaccinationCertificate
     ) {
         name.text = certificate.fullName
-        birthDate.text = getString(
-            R.string.vaccination_details_birth_date,
-            certificate.dateOfBirth.toDayFormat()
-        )
-        vaccinatedAt.text = certificate.vaccinatedAt.toDayFormat()
+        dateOfBirth.text = certificate.dateOfBirth.toString(dayOfBirthFormatter)
```

Comment:
I thin extension function for formatting birthdate would be great

Reply:
Yes, good point! I will add it